packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -327,6 +404,7 @@ export class ChildLogger extends TelemetryLogger {
 		baseLogger?: ITelemetryBaseLogger,
 		namespace?: string,
 		properties?: ITelemetryLoggerPropertyBags,
+		eventSamplingConfigs?: Map<string, SamplingConfig>,
```

Comment:
We'll want to add an `@param` block for this.

Reply:
It might also be worth extracting `Map<string, SamplingConfig>` into a type alias so we can shared documentation across usages.